netci.groovy

```diff
@@ -28,30 +25,35 @@ platformList.each { platform ->
     def buildCommand = '';
     def osForGHTrigger = os
     def version = "latest-or-auto"
+    def dockerRepository = "microsoft/dotnet-buildtools-prereqs"
+    def dockerContainer = ''
+    def dockerWorkingDirectory = "/src/core-setup"
+    def dockerCommand = ''
+    def portableArgs = ''
 
     // Calculate build command
     if (os == 'Windows_NT') {
         buildCommand = ".\\build.cmd -ConfigurationGroup=${configuration} -TargetArchitecture=${architecture}"
+        if ((architecture == 'arm' || architecture == 'arm64')) {
```

Comment:
Correct - we can't build deb packages on RHEL - which is where our Linux portable builds are run.

Also, I would think it is valuable to have at least 1 "non-portable" Linux CI leg to ensure we don't break "build from source" building non-portable assets.